src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let total = 0;
+
+  const getTotal = (value) => {
+    if (value !== undefined) {
+      total += value;
+
+      return getTotal;
+    } else {
+      const result = total;
+
```

Comment:
This line is unnecessary and can be removed for clarity. It doesn't contribute to the logic of the function.